def handle_error(error):
  print error.response.text
  return error
